YIF.Core.Domain/Repositories/LectorRepository.cs

```diff
@@ -0,0 +1,66 @@
+﻿using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using YIF.Core.Data.Entities;
+using YIF.Core.Data.Interfaces;
+using YIF.Core.Domain.DtoModels.EntityDTO;
+
+namespace YIF.Core.Domain.Repositories
+{
+    public class LectorRepository : ILectorRepository<Lecture, LectureDTO>
+    {
+        private readonly IApplicationDbContext _context;
+        private readonly IMapper _mapper;
```

Comment:
remove mapping from this class, because mapping isn`t responsibility of repository